xml/net_sync.xml

```diff
@@ -80,782 +74,369 @@
+   comment = Our backup target</screen>-->
+    </step>
+    <!--<step>
+     <para>Create the directory <filename>/var/backup</filename> belonging
+      to &rootuser;.</para>
+    </step>-->
+    <step>
+     <para>
+      Start and enable the rsyncd daemon with:
+     </para>
+     <screen>&prompt.root;<command>systemctl</command> enable rsyncd
+&prompt.root;<command>systemctl</command> start rsyncd</screen>
+     <!--<para>
+      <remark>taroth 2014-02-25: systemd - what about xinetd? is this
     still valid?</remark> - taroth 2014-03-17: for the records, here fcrozat's
     answer: xinetd is still there (although systemd can provide similar  functionalities
     but the configuration is different)
-   -->
-    syncd can alternatively be started by xinetd. This is, however, only
-    recommended for servers that rarely use rsyncd.
-   </para>
+      rsyncd can alternatively be started by xinetd. This is, however, only
+      recommended for servers that rarely use rsyncd.
+     </para> -->
+    </step>
+    <step>
+     <para>
+      Test the access to your Rsync server:
+     </para>
+     <screen>&prompt.user;<command>rsync</command> &wsI;::</screen>
+     <para>
+      It will respond with:
+     </para>
+     <screen>backup          Our backup target</screen>
+     <para>If not, check your configuration file, firewall and network settings.</para>
+    </step>
+   </procedure>
    <para>
+    The above steps create a Rsync server which can now be used to store
+    backups.
     The example also creates a log file listing all connections. This file is
-    stored in <filename>/var/log/rsyncd.log</filename>.
+    stored in <filename>/var/log/rsyncd.log</filename>. This is useful if you
+    want to debug your transfers.
    </para>
    <para>
-    It is then possible to test the transfer from a client system. Do this with
-    the following command:
+    To list all the content of your backup target, use the following command:
    </para>
-<screen>rsync -avz &exampleserver;::FTP</screen>
+<screen>rsync -avz &wsI;::backup</screen>
    <para>
     This command lists all files present in the directory
-    <filename>/srv/ftp</filename> of the server. This request is also logged in
+    <filename>/var/backup</filename> on the server. This request is also logged in
     the log file <filename>/var/log/rsyncd.log</filename>. To start an actual
-    transfer, provide a target directory. Use <literal>.</literal> for the
-    current directory. For example:
+    transfer, provide a source directory. Use <literal>.</literal> for the
+    current directory. For example, the following command copies the current
+    directory to your Rsync backup server:
    </para>
-<screen>rsync -avz &exampleserver;::FTP .</screen>
+   <screen>rsync -avz . &wsI;::backup</screen>
    <para>
-    By default, no files are deleted while synchronizing with rsync. If this
+    By default, no files are deleted while synchronizing with Rsync. If this
```

Comment:
By default, no files are deleted while synchronizing with Rsync

**By default, Rsync does not delete files and directories during synchronization**